pwndbg/aglib/kernel/slab.py

```diff
@@ -208,23 +216,25 @@ def freelist(self) -> Freelist:
     def active_slab(self) -> Slab | None:
         slab_key = slab_struct_type()
         _slab = self._cpu_cache[slab_key]
-        if not _slab:
+        if not int(_slab):
```

Comment:
Why we need this?

Reply:
you need explicit cast it to value, because `not _slab` is just object